### ISSUE TITLE
Button within focus trap that uses Clipboard js does not work

### ISSUE THREAD

**Issue by asaadmahmood** [focus-trap/focus-trap-react#51]
A button that uses clipboard js does not work if it exists within the focustrap.
https://clipboardjs.com/
